client/components/LayoutEditor/layoutEditor.scss

```diff
@@ -46,6 +47,13 @@
 			padding-top: 12px;
 		}
 	}
+	.dropdown-grouping {
+		display: flex;
+		// align-items: flex-end;
+		.dropdown {
+			margin: 0 0 5px 20px;
```

Comment:
I don't think any further `margin-bottom` is required here (but interested to hear your thoughts on it)

Reply:
it doesn't, I think it was a copy paste thing or maybe i freaked before adding the 20px to the left. its removed